test/test_torch.py

```diff
@@ -6092,6 +6093,7 @@ def test_pow(self, device):
                                                 r'Integers to negative integer powers are not allowed\.'):
                         torch.pow(m1[4], num)
                 else:
+                    expected_dtype = torch.result_type(num, m1[4])
```

Comment:
Why m1[4]? @ngimel and I thought m1 is a tensor?

Reply:
it's just the value that was being validated in existing tests (e.g. see above for `torch.pow(m1[4], num)`)

Reply:
Weird, but seems like not an issue for this change?